cinch/site.yml

```diff
@@ -31,6 +31,17 @@
   roles:
     - repositories
 
+- name: enable additional package streams on RHEL 8
+  become: true
+  hosts: repositories
+  tasks:
+    - name: enable libselinux-python stream
+      shell: dnf -y module enable libselinux-python
```

Comment:
These tasks should be guarded with some method of verifying that they've been executed and performed a change. The `shell` module allows you to check for the presence or absence of a particular filepath. Any task is able to check arbitrary conditions using the `changed_when`.

The best solution, though, is to either improve the `dnf` module upstream or create a separate module that can properly perform this task in an idempotent manner. Speaking of which, does the `@` notation work for this? I see the module page indicating installing a "modularity appstream" using the notation of `@postgresql/client`